modules/flowable-json-converter/src/main/java/org/flowable/editor/language/json/converter/ServiceTaskJsonConverter.java

```diff
@@ -154,6 +154,10 @@ protected void convertElementToJson(ObjectNode propertiesNode, BaseElement baseE
             if (serviceTask.isUseLocalScopeForResultVariable()) {
                 propertiesNode.put(PROPERTY_SERVICETASK_USE_LOCAL_SCOPE_FOR_RESULT_VARIABLE, serviceTask.isUseLocalScopeForResultVariable());
             }
+            
+            if(StringUtils.isNotEmpty(serviceTask.getFailedJobRetryTimeCycleValue())) {
```

Comment:
Please add spacing in `if(StringUtils...)` to match project standards; thanks

Reply:
Done